tests/chainer_tests/links_tests/model_tests/test_vision.py

```diff
@@ -259,13 +260,26 @@ def test_copy_gpu(self):
         self.check_copy()
 
 
+@testing.parameterize(*testing.product({
+    'dtype': [numpy.float16, numpy.float32],
+}))
 @unittest.skipUnless(googlenet.available, 'Pillow is required')
 @attr.slow
 class TestGoogLeNet(unittest.TestCase):
 
     def setUp(self):
+        config = chainer.config
```

Comment:
See this comment: https://github.com/chainer/chainer/pull/5114#discussion_r203917883

Reply:
I need to be good at Python more :(

Reply:
Fixed!